test_fms/fms2_io/test_atmosphere_io.sh

```diff
@@ -1,2 +1,12 @@
 #!/bin/sh
-bats ${srcdir}/test_fms/fms2_io/test_atmosphere_io.bats
+# This is part of the GFDL FMS package. This is a shell script to
+# execute tests in the test_fms/fms2_io directory.
+
+# Set common test settings.
+. ../test_common.sh
+
+# link to the input namelist
+cp ${srcdir}/test_fms/fms2_io/input_base.nml input.nml
```

Comment:
@wrongkindofdoctor In travis `make distcheck` fails because `cp: cannot create regular file 'input.nml': File exists` 

The test passes if you remove this line.